manifests/init.pp

```diff
@@ -90,7 +90,10 @@
 #                               disabled to let a dedicated sudo module manage it instead.
 #                               type:Boolean
 #
-# $use_sudoersd::               Add a file to /etc/sudoers.d (true) or uses augeas (false)
+# $use_sudoersd::               Add a file to /etc/sudoers.d (true).
+#                               type:Boolean
+#
+# $use_sudoers::                Add contents to /etc/sudoers (true).
```

Comment:
Will do - new commit points that out. It was already there in params.pp, so I only updated init.pp.